src/react/components/form.jsx

```diff
@@ -138,7 +138,7 @@ export default class Form extends React.Component {
   }
 
   getError = ref => {
-    return this.state.errors.get(ref);
+    return this.state.errors.get(ref) ? `ValidationError: ${this.state.errors.get(ref).message}` : this.state.errors.get(ref);
```

Comment:
Instead of returning `this.state.errors.get(ref)` in the false condition, it should return just `null` instead.  This way, it is clear what the function returns when there is no error.